### PR TITLE
Add Charsmap Operation

### PR DESCRIPTION
**Pull request by apaniukov**

### CODE REVIEW

File: src/charsmap_normalization.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#include "charsmap_normalization.hpp"
+#include "utils.hpp"
+#include "sentencepiece_trainer.h"  // for making normalizer spec
+#include "absl/strings/str_format.h"
+
+using namespace ov;
+
+namespace {
+    sentencepiece::NormalizerSpec make_identity_spec() {
+        return sentencepiece::SentencePieceTrainer::GetNormalizerSpec("identity");
```
Comment (ilya-lavrenov) [inline src/charsmap_normalization.cpp:14]: please, link `sentencepiece_train-static` to use train symbols (see https://github.com/google/sentencepiece/blob/d8f741853847553169444afc12c00f4bbff3e9ce/src/CMakeLists.txt#L226)

Comment (ilya-lavrenov) [reply src/charsmap_normalization.cpp:14]: Fixed in https://github.com/openvinotoolkit/openvino_tokenizers/pull/272

File: src/charsmap_normalization.hpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#pragma once
+
+#include "normalizer.h"  // from sentencepiece
+#include <openvino/op/op.hpp>
+
+
+using namespace ov;
+
+class CharsMapNormalization : public ov::op::Op {
```
Comment (pavel-esir) [inline src/charsmap_normalization.hpp:13]: Could you please add docstring with brief description what this operation does?

**Review by pavel-esir — APPROVED** [review]
in general LGTM, only one comment left

File: src/charsmap_normalization.hpp (hunk 2 above)
Comment (apaniukov) [reply src/charsmap_normalization.hpp:20]: Added

**Pull request merged**